torchinductor/config.py

```diff
@@ -49,6 +49,13 @@
 # replace small reductions with pointwise, disable with `= 1`
 unroll_reductions_threshold = 8
 
+# Cudagraphify will run even if we have graph mutations, may segfault
```

Comment:
I believe this will be silently incorrect, not segfault.

When would the user want this behavior?

Reply:
@anijain2305 asked the same thing above.

There are cases in prod models where we think we are over-enthusiastically deciding that a graph was mutated and then abort attempting cudagraphify. This just lets it always have a go. 

However, we are using fail/no fail as the indicator of success - not any correctness testing. Is there a way to check that what was produced is correct?

Reply:
The graph will return the correct answer, but the mutations to the inputs will be lost.

I might call this `force_cudagraphs` or something similar -- then update the comment to say it discards input mutations.

Reply:
Sounds good, it should feel relatively unsafe to use, and force fits the idea.